.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Build, test, and lint
+on:
+  push:
+    branches: [ dev, main ]
+  pull_request:
+    branches: [ dev, main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '16' ]
```

Comment:
@greatislander I don't think our build tools are compatible with Node 16 yet (sigh). See https://github.com/pressbooks/pressbooks-malala/runs/6281068108?check_suite_focus=true and https://github.com/pressbooks/pressbooks-build-tools/issues/283.

Reply:
@SteelWagstaff noted.

Reply:
I could be wrong about what was happening in those failed builds we were running into when I tried to bump node to 16 in other repos a while back though. I saw some weird stuff in build tools CI pipeline realted to peer dependencies and humanmade's eslint-config-react-app: https://github.com/pressbooks/pressbooks-build-tools/runs/7571285106?check_suite_focus=true that I was able to resolve by adding --legacy-peerd-deps in this commit: https://github.com/pressbooks/pressbooks-build-tools/pull/282/commits/876738ec0cfbb31db1bd94b051b5d48e36c7c6fc. See https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh. Not probably a great long term decision, I know, but I saw some promising discussion in humamade's repo about what they want to do: https://github.com/humanmade/coding-standards/issues/252. Maintaining and caring for the build tools repo itself is still a little over my head, tbh, but I'm trying to learn.